media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp

```diff
@@ -518,7 +518,14 @@ VAStatus MediaLibvaCapsG12::GetPlatformSpecificAttrib(VAProfile profile,
         }
         case VAConfigAttribPredictionDirection:
         {
-            *value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_FUTURE | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
+            if ((entrypoint == VAEntrypointEncSliceLP) &&
+                ((profile == VAProfileHEVCSccMain) ||
+                 (profile == VAProfileHEVCSccMain10) ||
+                 (profile == VAProfileHEVCSccMain444) ||
+                 (profile == VAProfileHEVCSccMain444_10)))
+                *value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
```

Comment:
@xhaihao Does it mean SCC only support low-delay-B?

Reply:
No, I don't think SCC has this requirement. but the driver requires l0 == l1, see https://github.com/intel/media-driver/blob/master/media_driver/agnostic/gen12/codec/hal/codechal_vdenc_hevc_g12.cpp#L2094-L2148